tests/Config/RedisConnectionConfigTest.php

```
@@ -125,28 +124,60 @@ public function testConstructorThrowsExceptionWhenRetryIntervalIsNegative(): voi
+                break;
+            case 'readTimeout':
+                $config = new RedisConnectionConfig('localhost', 6379, 2.5, null, 0, 'session:', false, 100, 0.0);
+                break;
+            case 'maxRetries':
+                $config = new RedisConnectionConfig('localhost', 6379, 2.5, null, 0, 'session:', false, 100, 2.5, 0);
+                break;
+            case 'retryInterval':
+                $config = new RedisConnectionConfig('localhost', 6379, 2.5, null, 0, 'session:', false, 0);
+                break;
+            default:
+                throw new \InvalidArgumentException("Unknown parameter: {$parameter}");
```

PHPMD (2.15.0)

Missing class import via use statement (line '147', column '27'). (undefined)

(MissingImport)